packages/sync/test/OfflineQueueLink.test.ts

```diff
@@ -98,6 +98,48 @@ describe("OnOffLink", () => {
     expect(testLink.operations.length).eqls(1);
   });
 
+  it("test online only directive skips queue", () => {
+    let operations: any[] = [];
+    const storageEngine = {
+      getItem() {
+        return operations;
+      },
+      removeItem() {
+        operations = [];
+      },
+      setItem(key: string, content: any) {
+        operations = JSON.parse(content);
+      }
+    };
+    const localConfig = { mutationsQueueName: "test", storage: storageEngine };
+    const queueLink = new QueueLink(localConfig);
+    queueLink.close();
+    const customLink = ApolloLink.from([queueLink, testLink]);
```

Comment:
[ProTip] We can use test link to see if this was forwarded. If not then we do not need test link at all.